### ISSUE TITLE
`http-api-gateway` and `db` endpoints for SPARQL

### ISSUE THREAD

**Issue by aaj3f** [fluree/db#543]
## Description

Given interest in targeting user groups coming from semantic/knowledge-graph communities, it's important that we allow use of actual SPARQL for interacting w/ Fluree dbs. This means `http-api-gateway` interfaces for submitting SPARQL queries, as well as vetting that the `db`'s SPARQL parser and endpoints are working with all current functionality

**Comment by cap10morgan** [comment]
I have adapted the v2 SPARQL->FQL code to v3, but it's hard to tell what all worked then and what was supposed to work but didn't. It certainly didn't support all of SPARQL (and likely never will). I also encountered some bugs in the v2 code's behavior that likely exist in v2 as well.

So I'm going to PR a fairly minimal version of this with some tests that demonstrate the kinds of queries that work and a few examples of some that don't.

SPARQL is a deep, deep rabbit hole. It would be helpful to define some specific types of queries (with examples) that we want to have working for this or the next iteration of this work.
